src/sentry/static/sentry/app/bootstrap.tsx

```diff
@@ -86,10 +86,10 @@ Sentry.init({
   ...window.__SENTRY__OPTIONS,
   integrations: getSentryIntegrations(hasReplays),
   tracesSampleRate,
-  _experiments: {useEnvelope: true},
-  async beforeSend(event) {
-    return normalizeTransactionName(appRoutes, event);
-  },
+});
+
+Sentry.addGlobalEventProcessor(async event => {
+  return normalizeTransactionName(appRoutes, event);
```

Comment:
@dashed Transaction still goes through event processors and this is the internal workaround for it.
EventProcessors are considered internal.

Reply:
Oh Interesting 👍